<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PacientesForm</class>
 <widget class="QWidget" name="PacientesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pacientes</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>71</y>
     <width>90</width>
     <height>301</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QPushButton" name="btn_editar">
      <property name="text">
       <string>Editar</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btn_eliminar">
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="btn_registrar">
      <property name="text">
       <string>Registrar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tblview_pacientes">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>71</y>
     <width>501</width>
     <height>400</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>565</width>
     <height>440</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>20</y>
     <width>400</width>
     <height>45</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="ledit_buscar"/>
    </item>
    <item>
     <widget class="QPushButton" name="btn_buscar">
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_refrescar">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>410</y>
     <width>72</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Refrescar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
